doc/requirements/io_class

```diff
@@ -0,0 +1,86 @@
+---
+group: IO class
+---
+
+Requirements for IO classes
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Setting occupancy limit
+id: occupancy
+---
+
+It shall be possible to set a maximum size of an IO class.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Allowed occupancy values
+id: occupancy_allowed_values
+---
+
+The product shall accept values for a max size of an IO class in range from
```

Comment:
maybe we can say that the product accepts integer and non-integer values in range... Then the occupancy_legacy_values req would not be needed.